gcp_census/bigquery/bigquery_task.py

```diff
@@ -50,17 +50,31 @@ def schedule_task_for_each_table(self, project_id, dataset_id,
         else:
             logging.info("There is no more tables in this dataset")
 
+    def schedule_task_for_each_partition(self, project_id, dataset_id, table_id,
+                                         partitions):
+        tasks = self.create_partition_tasks(project_id, dataset_id, table_id,
+                                            partitions)
+        Tasks.schedule(queue_name='bigquery-tables', tasks=tasks)
```

Comment:
I'm wondering if partition tasks should be handled on dedicated queue. It will probably give us better visibility and option to purge one of them without affecting others.

Reply:
I agree. I thought that having single queue would simplify throughput configuration, but separate queue seems to be a better option.